src/status_im/ui/screens/communities/reorder_categories.cljs

```diff
@@ -144,33 +152,35 @@
     :container-style      {:margin-bottom 108}          ;;        after bumping react native version to > 0.64 
     :on-drag-end-fn       on-drag-end-fn}])
 
+(defn chats_and_categories []
+  (let [{:keys [tab]} @state]
+    [:<>
+     [react/view {:flex-direction :row :margin 10 :border-radius 8 :background-color colors/blue-light}
```

Comment:
margin is different in designs
![image](https://user-images.githubusercontent.com/17097240/157874003-aaaf4bcb-af57-443f-afa9-3a5540fab162.png)